### DISCUSSION TITLE
I can not find the Ethereum Kovan in chainlink faucets  in lesson 4

### DISCUSSION THREAD

**Discussion by PratikKhaire** [Cyfrin/foundry-full-course-cu#155] [Q&A] [answered]
**### I am get issue at the lesson 4 of this course at the timestamp of video 4:30:50** 
i want to resolved this issue and help me 

Screenshot : - 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/109129126/aaf13353-f0bb-4e83-b294-98b16802fa8a)

Description : - 
as you see in above picture no kovan is present in it 

Step to open issue : - 
[Go to this link](https://faucets.chain.link/)

**Comment by ghost** [comment, marked as answer]
Hello @PratikKhaire the Kovan testnet was deprecated a long time ago, currently the recommended testnet is Sepolia.

**Reply by PratikKhaire** [reply]
Ok Thanks a lot @pacelliv 😊

**Reply by cromewar** [reply]
Marking this as answer, thanks @pacelliv
